pkg/tfgen/generate_nodejs_utilities.go

```diff
@@ -0,0 +1,37 @@
+package tfgen
+
+const tsUtilitiesFile = `
```

Comment:
I was surprised we aren't using the overlay system to allow this to be authored in normal `.ts` files.  Is that because overlays can only be provided via the end user of tfgen?

Reply:
> Is that because overlays can only be provided via the end user of tfgen?

Yes.